packages/node-firebird-driver/src/test/tests.ts

```diff
@@ -1,8 +1,43 @@
-import { Blob, Client, TransactionIsolation } from '../lib';
+import {Blob, Client, TransactionIsolation} from '../lib';
```

Comment:
?

Reply:
I think `prettier` and Git hooks (possible through husky) could be a great deal if configured as part of this project as they can ensure code style prior to commit. Here's an article of my own on this subject (https://marcioreis.pt/2019/09/bring-code-consistence-to-your-dev-team-projects/) 

This can minimize issues such as this one which was clearly caused by my IDE autoformat.

Reply:
I never saw a automatically formatted code base with readable code, unfortunatelly. For me readability (and consistency) is more important than consistency by consistency.